nym-vpn-core/crates/nym-vpn-lib/src/vpn/mixnet.rs

```diff
@@ -141,6 +141,14 @@ impl NymVpn<MixnetVpn> {
             entry_mixnet_gateway_ip,
             default_lan_gateway_ip,
         );
+
+        #[cfg(target_os = "android")]
+        {
+            if let Some(raw_fd) = mixnet_client.gateway_ws_fd().await {
+                self.android_tun_provider.clone().bypass(raw_fd);
```

Comment:
From what I remember, this is something we were doing before too, in a similar place. I think the idea of @pronebird is to move this kind of platform specific code into the same place, as to be easier to maintain and more stable. Maybe also check with him if this is the appropriate place to call the `bypass` function

Reply:
Yeah, at some point I think it was accidentally removed. Yeah, I agree it should be moved in the future. I can check, but this is essentially the same spot we had this before.